e2e/utils/index.ts

```diff
@@ -374,6 +374,68 @@ export function newProject({
   }
 }
 
+export function newLernaWorkspace({
+  name = uniq('lerna-proj'),
+  packageManager = getSelectedPackageManager(),
+} = {}): string {
+  try {
+    const projScope = name;
+    projName = name;
+
+    const pm = getPackageManagerCommand({ packageManager });
+
+    createNonNxProjectDirectory(projScope);
+
+    if (process.env.NX_VERBOSE_LOGGING == 'true') {
+      logInfo(`NX`, `E2E test has created a lerna workspace: ${tmpProjPath()}`);
+    }
+
+    // We need to force the real latest version of lerna to depend on our locally published version of nx
+    // TODO: translate this for yarn and pnpm
```

Comment:
I can look into cross package manager `overrides` in a follow up, the tests are not run against multiple package managers right now anyway because it is not important for the initial set of key tests